app/models/claim.rb

```diff
@@ -208,6 +210,7 @@ class Claim < ApplicationRecord
 
   validates :details_check, on: [:"teacher-detail"], inclusion: {in: [true, false], message: "Select an option to whether the details are correct or not"}
   validates :email_address_check, on: [:"select-email"], inclusion: {in: [true, false], message: "Select an option to indicate whether the email is correct or not"}
+  validates :mobile_check, on: [:"select-mobile"], presence: {message: "Select an option to indicate whether the mobile number is correct or not"}
```

Comment:
Is it also worth adding validation for `mobile_check` allowed values (`use`, `alternative`, `declined`) and a test for it

Reply:
Done, thanks!